### PR TITLE
Manual triggered:  Import config documentation from pump:master

### PR DESCRIPTION
**Pull request by Tyk-ITS**
Automated changes by [create-pull-request](https://github.com/peter-evans/create-pull-request) GitHub action

### CODE REVIEW

File: tyk-docs/content/shared/pump-config.md [hunk 1]
```diff
@@ -1675,9 +1675,9 @@ The topic that the writer will produce messages to.
 
 ### pumps.kafka.meta.timeout
 EV: <b>TYK_PMP_PUMPS_KAFKA_META_TIMEOUT</b><br />
-Type: `time.Duration`<br />
+Type: `interface{}`<br />
```
Comment (dcs3spp) [inline tyk-docs/content/shared/pump-config.md:1678]: Is this the correct type?

File: tyk-docs/content/shared/pump-config.md [hunk 2]
```diff
@@ -3081,6 +3081,13 @@ Type: `[]string`<br />
 
 Metrics to exclude from exposition. Currently, excludes only the base metrics.
 
+### pumps.prometheus.meta.track_all_paths
+EV: <b>TYK_PMP_PUMPS_PROMETHEUS_META_TRACKALLPATHS</b><br />
+Type: `bool`<br />
+
+Specifies if it should expose aggregated metrics for all the endpoints. By default, `false`
+which means that all APIs endpoints will be counted as 'unknown' unless the API use track endpoint plugin.
```
Comment (dcs3spp) [inline tyk-docs/content/shared/pump-config.md:3089]:
```suggestion
which means that all APIs endpoints will be counted as 'unknown' unless the API uses the track endpoint plugin.
```

**Review by dcs3spp — CHANGES_REQUESTED** [review]
@mativm02 @caroltyk For the Config documentation there is now a new workflow that should be used for generating automated content for configs etc.  There are checklist options to generate config for Pump and/or MDCB and merge to master.  
https://github.com/TykTechnologies/exp/actions/workflows/tyk-docs.yml  

I have reviewed the content on config files. Please use the new workflow to generate the config files for Pump with requested changes from the review comments.

File: tyk-docs/content/shared/pump-config.md (hunk 1 above)
Comment (mativm02) [reply tyk-docs/content/shared/pump-config.md:1678]: It is correct since we want to accept both time.Duration and int values now.

**Comment by dcs3spp** [comment]
@mativm02 Closing PR. PR approved in source repository https://github.com/TykTechnologies/tyk-pump/pull/722#pullrequestreview-1617798137

We just need the workflow config triggered now for new approved Tyk Pump Config using the new workflow GHub action

**Pull request closed without merging**